go-controller/pkg/ovn/pods.go

```diff
@@ -319,38 +327,62 @@ func (oc *Controller) addLogicalPort(pod *kapi.Pod) error {
 			}
 		}
 
-		addresses = podMac.String() + " " + util.JoinIPNetIPs(podIfAddrs, " ")
-		args = []string{
-			"--may-exist", "lsp-add", logicalSwitch, portName,
-			"--", "lsp-set-addresses", portName, addresses,
-		}
 	}
-	args = append(args, "--", "set", "logical_switch_port", portName, "external-ids:namespace="+pod.Namespace, "external-ids:pod=true")
-	args = append(args, "--", "lsp-set-port-security", portName, addresses)
 
-	out, stderr, err = util.RunOVNNbctl(args...)
+	// set addresses on the port
+	addresses = make([]string, len(podIfAddrs)+1)
```

Comment:
more idiomatic version:
```go
addresses = make([]string, 0, len(podIfAddrs) + 1)
addresses = append(addresses, podMac.String()
for _, addr := range podIfAddrs{
    addresses = append(addresses, podIfAddr.IP.String()
}
```

Reply:
apparently, a simple benchmark reveals that append is slower than array subscript. Hence this version.